### PR TITLE
fix event number in Digitizer

### PR DESCRIPTION
**Pull request by AllaMaevskaya**
bug fixes in Digitizer:
 

- event number; 
-  remove very slow particles (with TOF>250ns);
-  correct treatment of events with the same BC/orbit

### CODE REVIEW

File: Detectors/FIT/FT0/simulation/src/Digitizer.cxx [hunk 1]
```diff
@@ -28,6 +28,9 @@ ClassImp(Digitizer);
 
 Digitizer::CFDOutput Digitizer::get_time(const std::vector<float>& times, float deadTime)
 {
+  if (mEventID == 35) {
+    LOG(INFO) << " event 35 " << times.size() << " dead " << deadTime;
+  }
```
Comment (sawenzel) [inline Detectors/FIT/FT0/simulation/src/Digitizer.cxx:33]: Looks like debugging changes. Do you want to commit this?

File: Detectors/FIT/FT0/simulation/src/Digitizer.cxx [hunk 2]
```diff
@@ -122,9 +125,11 @@ Digitizer::CFDOutput Digitizer::get_time(const std::vector<float>& times, float
     }
   }
   if (!result.particle) {
-    LOG(DEBUG) << "CFD failed to find peak ";
-    for (auto t : times) {
-      LOG(DEBUG) << t << ", dead time " << deadTime;
+    if (times.size() > 104) {
```
Comment (sawenzel) [inline Detectors/FIT/FT0/simulation/src/Digitizer.cxx:128]: Looks like debugging changes. Do you want to commit this?

Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/simulation/src/Digitizer.cxx:128]: O, sorry, forget to remove

Comment (sawenzel) [reply Detectors/FIT/FT0/simulation/src/Digitizer.cxx:125]: I think this still needs to be removed. Or is there something particular about 104 times?

**Comment by AllaMaevskaya** [comment]
Hello
this magic "104 times" means that i want to know reason  why it was no CFD signal even if signal was big enough

File: Detectors/FIT/FT0/simulation/src/Digitizer.cxx (hunk 1 above)
Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/simulation/src/Digitizer.cxx]: I removed all additional debug printout

**Comment by AllaMaevskaya** [comment]
Dear Sandro
what is wrong now? I removed ( I hope) all un-necessary printouts

**Pull request merged**